test/fakeintake/server/server.go

```diff
@@ -97,8 +98,9 @@ func NewServer(options ...Option) *Server {
 		server: http.Server{
 			Addr: "0.0.0.0:0",
 		},
-		storeDriver:     "memory",
-		forwardEndpoint: "https://app.datadoghq.com",
+		storeDriver:        "memory",
+		forwardEndpoint:    "https://app.datadoghq.com",
+		logForwardEndpoint: "https://agent-http-intake.logs.datadoghq.com",
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
```suggestion
		// Source: https://docs.datadoghq.com/api/latest/logs/
		logForwardEndpoint: "https://agent-http-intake.logs.datadoghq.com",
```